web/fanfare/src/abi/index.ts

```diff
@@ -1,5 +1,5 @@
-import IERC20 from '@openzeppelin/contracts/build/contracts/IERC20.json';
+import ERC20 from '@openzeppelin/contracts/build/contracts/ERC20.json';
```

Comment:
This should be the interface that they expose instead of their implementation https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/interfaces/IERC20.sol

Reply:
IERC20's ABI is missing token metadata (even though it's de facto standard)

Reply:
Found one that includes it!

Reply:
IERC20Metadata instead then?